2022/wordle/german/sentence_translations.json

```diff
@@ -20,7 +20,7 @@
  {
   "input": "And what I thought I'd do here is just talk through with you some of my process in that, and explain some of the math that went into it, since the whole algorithm centers on this idea of entropy.",
   "model": "nmt",
-  "translatedText": "Und ich dachte, ich würde hier einfach mit Ihnen einige meiner Prozesse besprechen und einige der darin enthaltenen Mathematik erklären, da der gesamte Algorithmus auf dieser Idee der Entropie basiert.",
+  "translatedText": "Und was ich mir überlegt habe, ist dass ich mit euch ein Paar meiner Arbeitsschritte bespreche und die darin enthaltenen Mathematik erkläre, da der gesamte Algorithmus auf dieser Idee der Entropie basiert.",
```

Comment:
That's a fundamental discussion point Grant needs to decide upon: stick with the polite "Sie" form from the initial translation (I did that in my proposals) or change that to ihr?